rasa/validator.py

```diff
@@ -132,18 +132,18 @@ def _gather_utterance_actions(self) -> Set[Text]:
         """Return all utterances which are actions."""
         return {
             utterance
-            for utterance in self.domain.templates.keys()
+            for utterance in self.domain.responses.keys()
             if utterance in self.domain.action_names
         }
 
     def verify_utterances(self, ignore_warnings: bool = True) -> bool:
         """Compares list of utterances in actions with utterances in responses."""
 
         actions = self.domain.action_names
-        utterance_templates = set(self.domain.templates)
+        utterance_responses = set(self.domain.responses)
```

Comment:
seems easier to me, what do you think?
```suggestion
        responses = set(self.domain.responses)
```

Reply:
Looking at `verify_utterances` and `verify_utterances_in_stories`, I'm a bit confused. Is there a separate list of utterances in actions compared to responses?

So this statement `The utterance {utterance} is not listed under actions in the domain file. It can only be used as a template`. What is template referring to here?

Reply:
The first check in lines 146-152 seems outdated and isn't required any longer. I think this is relict from the times when you had to define a responses in your `actions:` and below `responses:`. In my opinion we can just drop this code.